package.json

```
@@ -16,6 +16,7 @@
         "url": "https://github.com/grammyjs/hydrate/issues"
     },
     "scripts": {
+        "prepare": "npm run build",
         "build": "deno2node tsconfig.json",
```

```suggestion
        "build": "deno2node",
```
for forward compatibility with Deno2node 2.0.